Standards/scs-0212-v1-requirements-for-container-registries.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Requirements for container registries
+type: Standard
+status: Draft
+track: KaaS
+---
+
+## Introduction
+
+A container registry is an infrastructure service to enable storing and accessing container
+images. Images can be pushed to the registry by e.g. Continuous integration pipelines and
+be pulled from by runtime environments like Kubernetes clusters.
+
+Container registries come in various forms, e.g. publicly accessible ones like Docker Hub or
+self-hosted and cloud-hosted services. The later examples may apply various access control
```

Comment:
Here I think "latter" instead of "later" is meant, right?

Reply:
Fixed that commit.

Reply:
Changed in https://github.com/SovereignCloudStack/standards/pull/458/commits/e73ed463c4d03e128dcfa7f1437bfa7131de3f93